packages/functional-tests/pages/relier.ts

```diff
@@ -75,7 +75,7 @@ export class RelierPage extends BaseLayout {
   }
 
   async promptNoneError() {
-    await this.page.waitForSelector('.error');
+    this.page.locator('.error');
     return this.page.innerText('.error');
```

Comment:
Just because I was reading the [`Locator#innerText` docs](https://playwright.dev/docs/api/class-locator#locator-inner-text) a minute ago while looking up the details for `.waitFor()`:


> [!WARNING]
> **ASSERTING TEXT**
> If you need to assert text on the page, prefer [`expect(locator).toHaveText()`](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-text) with `useInnerText` option to avoid flakiness. See [assertions guide](https://playwright.dev/docs/test-assertions) for more details.

Although&hellip; now I think I'm possibly reading this all wrong anyways since it looks like we're using **`Page#innerText()`** and not `Locator#innerText()`. Hhhmm. Per [`Page#innerText()` docs](https://playwright.dev/docs/api/class-page#page-inner-text):

> [!WARNING]
> **DISCOURAGED**
> Use locator-based [`locator.innerText()`](https://playwright.dev/docs/api/class-locator#locator-inner-text) instead. Read more about [locators](https://playwright.dev/docs/locators).